partials/welcome-page.php

```diff
@@ -4,11 +4,11 @@
 		<div class="edac-welcome-header">
 			<div class="edac-welcome-header-right">
 				<a href="https://equalizedigital.com/" target="_blank">
-					<img src="<?php echo plugin_dir_url( __DIR__ ); ?>assets/images/edac-logo.png" alt="Equalize Digital">
+					<img src="<?php echo plugin_dir_url( __DIR__ ); ?>assets/images/edac-logo.png" alt="Equalize Digital Logo">
 				</a>
 			</div>
 			<div class="edac-welcome-header-left">
-				<h2 class="edac-welcome-title">
+				<h1 class="edac-welcome-title">
```

Comment:
@SteveJonesDev @alexstine  I expect we might need to have some CSS changes to support these bumped-up heading levels to ensure that the headings are not too big now.